contracts/mesh-consumer/src/ibc.rs

```diff
@@ -116,27 +117,50 @@ pub fn ibc_packet_receive(
     }
 }
 
-pub fn receive_list_validators(_deps: DepsMut) -> Result<IbcReceiveResponse, ContractError> {
-    // TODO
-    unimplemented!();
+pub fn receive_list_validators(deps: DepsMut) -> Result<IbcReceiveResponse, ContractError> {
+    let validators = deps.querier.query_all_validators()?;
```

Comment:
You need to convert to the proper ibc type.
 `mesh_ibc::ListValidatorsResponse`. I think: 

Also, see mesh-provider receive functions

```rust
let validators = validators.into_iter().map(|x| x.address).collect();
let ack = StdAck::success(mesh_ibc::ListValidatorsResponse { validators })?;
```